rdrf/report/schema.py

```diff
@@ -461,9 +470,13 @@ def clinical_data_resolver(patient, _info):
             collection="cdes"
         ).order_by('created_at').all()
 
+    def email_notification_resolver(patient, _info):
```

Comment:
You can resolve the EmailPreference object to be consumed by the DjangoObjectType  from here
e.g. `return EmailPreference.objects.get_by_user(patient.user)`